packages/backend/src/services/SpaceService/SpaceService.test.ts

```diff
@@ -0,0 +1,865 @@
+                },
+                {
+                    name: 'project admin can manage space even with multiple group roles',
+                    user: { projectRole: ProjectMemberRole.ADMIN },
+                    space: { isPrivate: true },
+                    access: {
+                        groupSpaceRoles: [
+                            SpaceMemberRole.VIEWER,
+                            SpaceMemberRole.EDITOR,
+                        ],
+                    },
+                    action: 'manage',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'user with multiple group roles gets highest role (editor over viewer)',
+                    user: { projectRole: ProjectMemberRole.INTERACTIVE_VIEWER },
+                    space: { isPrivate: true },
+                    access: {
+                        groupSpaceRoles: [
+                            SpaceMemberRole.VIEWER,
+                            SpaceMemberRole.EDITOR,
+                        ],
+                    },
+                    action: 'update',
+                    expectedResult: true,
+                    contentType: 'Dashboard',
+                },
+                {
+                    name: 'user with multiple group roles gets highest role (admin over editor)',
+                    user: { projectRole: ProjectMemberRole.INTERACTIVE_VIEWER },
+                    space: { isPrivate: true },
+                    access: {
+                        groupSpaceRoles: [
+                            SpaceMemberRole.EDITOR,
+                            SpaceMemberRole.ADMIN,
+                        ],
+                    },
+                    action: 'update',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'project group admin role overrides space viewer role',
+                    user: {
+                        projectRole: ProjectMemberRole.INTERACTIVE_VIEWER,
+                        projectGroupRoles: [ProjectMemberRole.ADMIN],
+                    },
+                    space: { isPrivate: true },
+                    access: { spaceRole: SpaceMemberRole.VIEWER },
+                    action: 'update',
+                    expectedResult: true,
+                    contentType: 'Space',
+                },
+                {
+                    name: 'project group viewer role does not override space editor role',
+                    user: {
+                        projectRole: ProjectMemberRole.INTERACTIVE_VIEWER,
+                        projectGroupRoles: [ProjectMemberRole.VIEWER],
+                    },
+                    space: { isPrivate: true },
+                    access: { spaceRole: SpaceMemberRole.EDITOR },
+                    action: 'update',
+                    expectedResult: true,
```

Comment:
this is still true, so the test is either wrong, or we can still update because INTERACTIVE_VIEWER.